relay-server/src/actors/envelopes.rs

```diff
@@ -1889,6 +1895,8 @@ impl Handler<ProcessMetrics> for EnvelopeProcessor {
 
                     relay_log::trace!("merging metric buckets into project cache");
                     project_cache.do_send(MergeBuckets::new(public_key, buckets));
+                } else {
+                    metric!(counter(RelayCounters::MetricBucketsParsingFailed) += 1);
```

Comment:
@jan-auer you had an argument against emitting statsd metrics here, do you remember what it was?